app/models/foreman_openscap/policy.rb

```diff
@@ -28,7 +29,7 @@ class Policy < ApplicationRecord
               :if => Proc.new { |policy| policy.should_validate?('Schedule') }
 
     validates :scap_content_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
-    validates :scap_content_profile_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
+    #validates :scap_content_profile_id, presence: true, if: Proc.new { |policy| policy.should_validate?('SCAP Content') }
```

Comment:
Maybe uncomment this and fix the default profiles separately? It should turn the tests green.